lightly_bucket/.lightly/metadata/KS-FR-BLOIS/24330/camcourt1_1513714448590_0.json

```diff
@@ -0,0 +1 @@
+{"file_name": "KS-FR-BLOIS/24330/camcourt1_1513714448590_0.png", "type": "image", "metadata": {"court": "KS-FR-BLOIS", "game_id": "24330"}}
```

Comment:
When dumping the metadata as json files I suggest using the `indent=4` parameter to format them more nicely.

E.g. 
```python
with open(path_to_prediction, 'w') as f:
    json.dump(lightly_prediction, f, indent=4) 
```